doc/sphinx/source/framework-integration-guides.rst

```diff
@@ -46,55 +46,31 @@ system that already has an ``ngraph_dist`` installed.
    .. code-block:: console
 
       $ sudo apt-get install python3-pip python3-venv
+      $ python3 -m venv .venv
+      (.venv)$ . bin/activate
 
+#. Go to the "python" subdirectory of the ``ngraph`` repo and complete the 
+   steps needed to get the wrapper installed: 
 
-#. Clone the ngraph-neon integration repo somewhere you can find it easily. The
-   install configuration we'll document here places it under a directory we'll 
-   create named ``frameworks``
-
-   * This section is optional, but it is the configuration we'll assume for the 
-     ``ngraph-neon`` documenation:
-
-      .. code-block:: console
+   .. code-block:: console
 
-         $ sudo mkdir -p /opt/frameworks   
-         $ sudo chown -R username:username /opt/frameworks
-         $ cd /opt/frameworks
+      (.venv)$ cd ngraph/python
+      (.venv)$ git clone --recursive -b allow-nonconstructible-holders https://github.com/jagerman/pybind11.git
+      (.venv)$ pip install -U . 
 
-   * This section is required:       
+#. Finally are we ready to install the `neon` integration: 
```

Comment:
typo...we are* instead of are we

Reply:
This is actually a language thing; I did intend it to be that way.  Varying sentence structure so reading instructions is not boring.  Because, seriously.  By the time people get this far through the instructions, they are going to be thinking *finally!*

Reply:
Compromise: `Finally our neon integration is ready to install.`